lib/rules/func-call-spacing.js

```diff
@@ -131,6 +131,14 @@ module.exports = {
                          * https://github.com/eslint/eslint/issues/7787
                          */
                         if (!hasNewline) {
+                            if (sourceCode.commentsExistBetween(leftToken, rightToken)) {
+                                const comments = sourceCode.getCommentsBefore(rightToken);
+
+                                return [
+                                    fixer.removeRange([leftToken.range[1], comments[0].range[0]]),
+                                    fixer.removeRange([comments[comments.length - 1].range[1], rightToken.range[0]])];
```

Comment:
It wouldn't fix:

```js
/*eslint func-call-spacing: ["error"]*/

f/**/ /**/();
```

Maybe we could do something like this:

https://github.com/eslint/eslint/blob/6cfbd03b3f22edb4d1c9c61c64eea7c129da71aa/lib/rules/space-before-function-paren.js#L135-L146

(that check for `"Line"` comment isn't necessary in this branch)